examples/apply-setters/simple/.expected/results.yaml

```diff
@@ -0,0 +1,6 @@
+name: apply-setters
+items:
+- message: 'set field value to "my-new-map" in file "resources.yaml" in field "metadata.name"'
```

Comment:
I saw you set Field and File in the code below, but why structured result doesn't show field and file?
IIRC there were some bug in the e2e test harness that it doesn't handle `results.yaml` when the exit code is non-zero.
I tried to update it in https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/275/files#diff-07257d5504beba4b3c7dd901580094eb1b7d3611b50ce944a09107c942b0bfb8R5

Reply:
Yes. That's a known issue. https://github.com/GoogleContainerTools/kpt/issues/1910

I deleted this file for now. We can upgrade the runner version we are using and add these back. There are other tests as well which are broken which can be fixed with upgrade.

Reply:
Yes. This is a known issue https://github.com/GoogleContainerTools/kpt/issues/1910

We should update it in the runner util in kpt, and pull it so that even other tests can also be fixed.